### PR TITLE
adding init.d scripts

### LINKED ISSUE #24: init.d scripts

### PR DESCRIPTION
**Pull request by burtonjc**
closes #24

### CODE REVIEW

**Comment by homeyer** [comment]
add instructions for running on osx in a virtualbox

File: bin/build.sh [hunk 1]
```diff
@@ -39,15 +43,13 @@ docker save --output="waffleio-takeout/waffle.io-hooks.tar" quay.io/waffleio/waf
 docker save --output="waffleio-takeout/waffle.io-migrations.tar" quay.io/waffleio/waffle.io-migrations
 docker save --output="waffleio-takeout/waffle.io-rally-integration.tar" quay.io/waffleio/waffle.io-rally-integration
 
-cp install.sh waffleio-takeout/
-cp waffleio-env.list waffleio-takeout/
+cp bin/install.sh waffleio-takeout/
```
Comment (homeyer) [inline bin/build.sh]: do we want to `cp`, or `move`? Could it be confusing to have copies that diverge?

File: bin/install.sh [hunk 2]
```diff
@@ -341,6 +346,20 @@ echo WEB_CONCURRENCY=$webConcurrency >> $envFile
 
 echo -e "\n\nGreat! That's all we need, we will have your Waffle.io Takeout ready shortly."
 
+######################################################
+# Store off environment configuration in /etc/waffle #
+######################################################
+echo -e "\n\nSaving environment configuration in /etc/waffle/"
+sudo mkdir -p /etc/waffle/ca-certificates
+sudo cp $envFile /etc/waffle/environment.list
```
Comment (homeyer) [inline bin/install.sh]: wondering what the best solution is... now we have 3 copies, and we overwrite this file on each install. Should we check for existence first? Should we copy sooner, and only update the one in /etc/waffle?

File: init.d/dockerized-local-service [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    ;;
+
+  # upgrade)
+  #   call_fn 'update_image'
+  #   call_fn 'restart'
+  #   ;;
+
+  *)
+    echo "Usage: ${0} {start|stop|status|reload|restart}"
+    # echo "Usage: ${0} {start|stop|status|reload|restart|upgrade}"
```
Comment (homeyer) [inline init.d/dockerized-local-service]: remove

File: doc/DEV.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  - Attached to: "Bridged Adapter"
+  - Name: Wi-Fi (AirPort)
+7. Under the "Shared Folders" tab, add a new Machine Folder to the location you want to keep the takeout files on your local machine. I chose ~/Desktop/waffle-takeout. You will have to create the folder on your machine first in order to share it with the virtual machine.
+  - Read-only: false
+  - Auto-mount: true
+  - Make Permanent: true
+8. Cick OK at the bottom to save your configurations
+9. Double click on the VirtualBox in the left column. It will boot and walk you through the installation process for Ubuntu.
+10. After installing, while in your VM, navigate to the top menu bar of OSX and select Devices > Insert Guest Additions CD.
+11. Inside the VM, you should see a window popup asking fi you want to run the software on the CD. Choose "Run" and walk through the process. If it doesn't automatically run, you should be able to run it from the CD icon on the sidebar within the VM.
```
Comment (homeyer) [inline doc/DEV.md]: `fi` -> `if`

File: bin/build.sh (hunk 1 above)
Comment (burtonjc) [inline bin/build.sh]: If we move, you would be unable to run the install more than once. Seems not great.

File: bin/install.sh (hunk 2 above)
Comment (burtonjc) [inline bin/install.sh]: I'm not sure what the best approach is yet. We need to account for both a clean install and updating an existing install. Namely, what if the user has set some environment variables already, but we need to add some, change some, and/or default some. Not super obvious what the right answer is.

Comment (burtonjc) [inline bin/install.sh]: To clarify, leaving `./environment.list` around as it is now allows the install to run multiple times without having to answer the hard question above.

File: doc/DEV.md (hunk 4 above)
Comment (burtonjc) [inline doc/DEV.md]: to much bash scripting...**cough**

**Comment by homeyer** [comment]
:+1:

**Pull request merged**